doc/web-types.md

```diff
@@ -0,0 +1,152 @@
+# Web-types
+
+## web-component-analyzer usage
+
+### With package.json configuration
+
+You can add a `wca` section in your `package.json` to configure wca for web-types build.
+
+```json
+"wca": {
+  "webtypesConfig": {
+    "name": "your-project-name",
+    "version": "0.0.1",
+    "framework": "lit",
+    "description-markup": "markdown"
+  }
+}
+```
+
+To run wca then use command:
+
+```bash
+wca analyze src --format webtypes --outFile web-types-custom.json
+```
+
+If you run `wca` from project npm `scripts` section of `package.json`, you can omit `name` and `version` property
+in `webtypesConfig`, this will take package `name` and `version` by default.
+
+### With command line only
+
+You can also avoid updating `package.json` `wca` section and use only command line by providing a json configuration to `--webtypesConfig` argument:
+
+```bash
+wca analyze src --format webtypes --outFile web-types-custom.json --webtypesConfig='{"name": "your-project-name", "version": "0.0.1", "framework": "lit", "description-markup": "markdown"}'
+```
+
+If you run `wca` from project npm `scripts` section of `package.json`, you can omit `name` and `version` property
+in `webtypesConfig`, this will take package `name` and `version` by default.
+
+## Project setup
+
+### For Lit
+
+Generated web-types are working with the generic `lit-web-types` library. You need to add it on your project.
+
+<!-- prettier-ignore -->
+```bash
+npm i lit-web-types -D
```

Comment:
@piotrtomiak 
Should I just update with `@web-types/lit` package here ?
```suggestion
npm i @web-types/lit -D
```
Or should I expect an IDE auto detection or something else ?

Reply:
The IDE will download web-types automatically if you have `lit` in dependencies, so there is no need to add the package to `package.json`

Reply:
I opened an issue to talk about that: https://github.com/JetBrains/web-types/issues/56